Product/Production/Adapters/General/CONNECTDirectConfig/src/main/java/gov/hhs/fha/nhinc/directconfig/dao/impl/TrustBundleDaoImpl.java

```diff
@@ -323,7 +328,7 @@ public void updateLastUpdateError(long trustBundleId, Calendar attemptTime, Bund
         } catch (ConfigurationStoreException cse) {
             DaoUtils.rollbackTransaction(tx);
             throw cse;
-        } catch (Exception e) {
+        } catch (HibernateException e) {
             DaoUtils.rollbackTransaction(tx);
             throw new ConfigurationStoreException("Failed to update bundle last refresh error.", e);
```

Comment:
Can you update this to be consistent?
throw new ConfigurationStoreException("Failed to update Trust Bundle Anchors: " + e.getLocalizedMessage(), e)